ui/server/src/main/scala/pl/touk/nussknacker/ui/api/NodesResources.scala

```diff
@@ -0,0 +1,48 @@
+package pl.touk.nussknacker.ui.api
+
+import akka.http.scaladsl.marshalling.ToResponseMarshallable
+import akka.http.scaladsl.model.{HttpResponse, StatusCodes}
+import akka.http.scaladsl.server.Route
+import de.heikoseeberger.akkahttpcirce.FailFastCirceSupport
+import pl.touk.nussknacker.engine.ProcessingTypeData
+import pl.touk.nussknacker.engine.additionalInfo.{NodeAdditionalInfo, NodeAdditionalInfoProvider}
+import pl.touk.nussknacker.engine.canonicalgraph.CanonicalProcess
+import pl.touk.nussknacker.engine.graph.node.NodeData
+import pl.touk.nussknacker.engine.util.loader.ScalaServiceLoader
+import pl.touk.nussknacker.ui.process.repository.FetchingProcessRepository
+import pl.touk.nussknacker.ui.security.api.LoggedUser
+import pl.touk.nussknacker.ui.process.ProcessingTypeDataProvider
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class NodesResources(val processRepository: FetchingProcessRepository[Future], typeToConfig: ProcessingTypeDataProvider[ProcessingTypeData])(implicit val ec: ExecutionContext)
```

Comment:
Will be nice to see some tests for this ;)

Reply:
I added tests